src/PointwiseFunctions/AnalyticSolutions/Elasticity/CMakeLists.txt

```diff
@@ -22,7 +23,9 @@ spectre_target_headers(
 target_link_libraries(
   ${LIBRARY}
   PUBLIC DataStructures
+  PUBLIC Integration
   INTERFACE ErrorHandling
   INTERFACE Utilities
   INTERFACE ConstitutiveRelations
```

Comment:
Make these three `PUBLIC` instead of `INTERFACE` (all three are used in source as well as header files)